src/main/java/bytetalker/task/TaskType.java

```diff
@@ -0,0 +1,34 @@
+package bytetalker.task;
+
+/**
+ * Represents the types of task supported by the chatbot.
+ *
+ * @author Junseo Kim
+ * @version 1.0
+ * @since 2024-02-06
+ */
+public enum TaskType {
+    TODO("T") {
+    },
+    DEADLINE("D") {
+
```

Comment:
Maybe you can standardise the formatting for the enum here. The first object "TODO" has no empty line after the curly bracket but "DEADLINE" and "EVENT" has one.

Reply:
Oh, alright I will try to standardise the format. Thank you for feedback!